corehq/apps/export/models/new.py

```diff
@@ -721,6 +753,37 @@ def merge(cls, one, two):
         item.options = options
         return item
 
+    def split_header(self, header, ignore_unspecified_options):
+        header_template = header if '{option}' in header else u"{name} | {option}"
+        headers = []
+        for option in self.options:
+            headers.append(
```

Comment:
yeah, definitely prefer that approach with more heavy weight things, but felt like this was sufficiently light weight